setup.py

```diff
@@ -32,20 +32,29 @@
 
 import os
 import pip
+import sys
 from pip.req import parse_requirements
 from setuptools import setup, find_packages
 
 
 requirements = [str(requirement.req) for requirement in parse_requirements(
     'requirements.txt', session=pip.download.PipSession())]
 
+# Python Version check
+if not sys.version_info[0] == 2:
+    sys.exit('Python 3 is not supported')
```

Comment:
What will be code changes needed for supporting python 3? Can we just issue a warning that it is untested on python 3, so that the installation is not prohibited? Dell 3000 series comes with python 3 by default, and i think we should not force users to go with an older version.